.storybook/main.ts

```diff
@@ -0,0 +1,25 @@
+import { TsconfigPathsPlugin } from 'tsconfig-paths-webpack-plugin';
```

Comment:
this makes sure out `baseUrl` setup from tsconfig is respected and we don't need to do the magic with babel